proto/concept.proto

```diff
@@ -0,0 +1,963 @@
+
+    message UnsetPlays {
+        message Req {
+            RoleType role_type = 1;
+        }
+        message Res {}
+    }
+
+    message GetSyntax {
+        message Req {}
+        message Res {
+            string syntax = 1;
+        }
+    }
+}
+
+message EntityType {
+    string label = 1;
+    bool is_root = 2;
+    bool is_abstract = 3;
+
+    message Create {
+        message Req {}
+        message Res {
+            Entity entity = 1;
+        }
+    }
+
+    message GetSupertype {
+        message Req {}
+        message Res {
+            EntityType entity_type = 1;
+        }
+    }
+
+    message SetSupertype {
+        message Req {
+            EntityType entity_type = 1;
+        }
+        message Res {}
+    }
+
+    message GetSupertypes {
+        message Req {}
+        message ResPart {
+            repeated EntityType entity_types = 1;
+        }
+    }
+
+    message GetSubtypes {
+        message Req {
+            Type.Transitivity transitivity = 1;
+        }
+        message ResPart {
+            repeated EntityType entity_types = 1;
+        }
+    }
+
+    message GetInstances {
+        message Req {
+            Type.Transitivity transitivity = 1;
+        }
+        message ResPart {
+            repeated Entity entities = 1;
+        }
+    }
+}
+
+message RelationType {
+    string label = 1;
+    bool is_root = 2;
+    bool is_abstract = 3;
+
+    message Create {
+        message Req {}
+        message Res {
+            Relation relation = 1;
+        }
+    }
+
+    message GetSupertype {
+        message Req {}
+        message Res {
+            RelationType relation_type = 1;
+        }
+    }
+
+    message SetSupertype {
+        message Req {
+            RelationType relation_type = 1;
+        }
+        message Res {}
+    }
+
+    message GetSupertypes {
+        message Req {}
+        message ResPart {
+            repeated RelationType relation_types = 1;
+        }
+    }
+
+    message GetSubtypes {
+        message Req {
+            Type.Transitivity transitivity = 1;
+        }
+        message ResPart {
+            repeated RelationType relation_types = 1;
+        }
+    }
+
+    message GetInstances {
+        message Req {
+            Type.Transitivity transitivity = 1;
+        }
+        message ResPart {
+            repeated Relation relations = 1;
+        }
+    }
+
+    message GetRelates {
+        message Req {
+            Type.Transitivity transitivity = 1;
+        }
+        message ResPart {
+            repeated RoleType role_types = 1;
+        }
+    }
+
+    message GetRelatesForRoleLabel {
+        message Req {
+            string label = 1;
+        }
+        message Res {
+            optional RoleType role_type = 1;
+        }
+    }
+
+    message GetRelatesOverridden {
+        message Req {
+            string label = 1;
+        }
+        message Res {
+            optional RoleType role_type = 1;
+        }
+    }
+
+    message SetRelates {
```

Comment:
I wonder if this should be `createRelates/deleteRelates` since we don't create RoleType any other way, is that right?

Bit tricky since it should be `createRole` and `deleteRole` or something like that in reality. 

We can leave it for now since it's no different to before. Worth a thought though.